resources/contribute-documentation/markdoc-tags.md

```diff
@@ -0,0 +1,265 @@
+---
+seo:
+    description: Read about all the Markdoc tags commonly used in XRPL documentation, including custom tags.
+---
+# Markdoc Tags
+
+This page serves as a reference of the [Markdoc tags](https://redocly.com/docs/realm/content/markdoc-tags/tag-library) used in XRPL documentation. These extensions of Markdown syntax allow for stylized and dynamic contents.
+
+When using self-closing tags, be sure to include the closing slash in the tag definition. Otherwise, the unclosed tag may accidentally hide all contents of the page after it.
```

Comment:
Suggest simplifying the last sentence for translation:

```suggestion
When using self-closing tags, be sure to include the closing slash in the tag definition. Otherwise, content placed after this unclosed tag will not be displayed.
```